src/game/ingame_menu.c

```diff
@@ -1523,18 +1512,27 @@ void render_pause_red_coins(void) {
     }
 }
 
-/// By default, not needed as puppycamera has an option, but should you wish to revert that, you are legally allowed.
+LangArray textCurrRatio43 = DEFINE_LANGUAGE_ARRAY(
+    "ASPECT RATIO: 4:3\nPRESS L TO SWITCH",
+    "RATIO D'ASPECT: 4:3\nAPPUYEZ SUR L POUR CHANGER",
+    "SEITENVERHÄLTNIS: 4:3\nDRÜCKE L ZUM WECHSELN",
+    "アスペクトひ: ４:３\nＬボタンできりかえ");
 
+LangArray textCurrRatio169 = DEFINE_LANGUAGE_ARRAY(
+    "ASPECT RATIO: 16:9\nPRESS L TO SWITCH",
+    "RATIO D'ASPECT: 16:9\nAPPUYEZ SUR L POUR CHANGER",
+    "SEITENVERHÄLTNIS: 16:9\nDRÜCKE L ZUM WECHSELN",
+    "アスペクトひ: １６:９\nＬボタンできりかえ");
+
+/// By default, not needed as puppycamera has an option, but should you wish to revert that, you are legally allowed.
 #if defined(WIDE) && !defined(PUPPYCAM)
 void render_widescreen_setting(void) {
     gSPDisplayList(gDisplayListHead++, dl_ia_text_begin);
     gDPSetEnvColor(gDisplayListHead++, 255, 255, 255, gDialogTextAlpha);
     if (!gConfig.widescreen) {
-        print_generic_string(10, 20, textCurrRatio43);
-        print_generic_string(10,  7, textPressL);
+        print_generic_string(10, 20, LANG_ARRAY(textCurrRatio43));
```

Comment:
Uhh not really related to the PR since that's how it has always been but sure it could be changed before merging. However remember to merge Spanish into this branch first